geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/StartMemberUtils.java

```diff
@@ -122,11 +124,26 @@ static void addGemFireSystemProperties(final List<String> commandLine,
     }
   }
 
+  private static final String[] REQUIRED_OPENS = new String[]{};
+  private static final String[] REQUIRED_EXPORTS = new String[]{
+      "java.base/sun.nio.ch",
+      "java.management/com.sun.jmx.remote.security",
+  };
+  private static final List<String> OPENS = Stream.of(REQUIRED_OPENS)
+      .map(s -> "--add-opens=" + s + "=ALL-UNNAMED")
+      .collect(toList());
+
+  private static final List<String> EXPORTS = Stream.of(REQUIRED_EXPORTS)
+      .map(s -> "--add-exports=" + s + "=ALL-UNNAMED")
+      .collect(toList());
+
   static void addJvmArgumentsAndOptions(final List<String> commandLine,
       final String[] jvmArgsOpts) {
     if (jvmArgsOpts != null) {
       commandLine.addAll(Arrays.asList(jvmArgsOpts));
     }
+    commandLine.addAll(OPENS);
```

Comment:
I know this is a draft but remember that these options (opens and exports) should only be added on JVMs that support those options. JDK 8 does not so this will break any tests that use this class on jdk 8.
Also consider putting these in a new method instead of overloading addJvmArgumentsAndOptions

Reply:
Yeah, I know. I should have marked the branch as a spike not wip.